RELEASING.md

```diff
@@ -61,8 +61,7 @@ of the Cardano Engineering Handbook for more detailed information on this topic.
 
 ### `CHANGELOG.md`
 
-Every package, that [is subject to the release
-process](packages-excluded-from-release-process), will always have the top most entry in
+Every package that [is subject to the release process](https://github.com/IntersectMBO/cardano-ledger/blob/master/RELEASING.md#packages-excluded-from-release-process) will always have the top most entry in
```

Comment:
I think it's better to use a local link, in case the location of the file changes:

```suggestion
Every package that [is subject to the release process](#packages-excluded-from-release-process) will always have the top most entry in
```